flutter/android/src/main/kotlin/io/sentry/flutter/SentryFlutterPlugin.kt

```diff
@@ -98,6 +100,7 @@ class SentryFlutterPlugin : FlutterPlugin, MethodCallHandler {
       args.getIfNotNull<Boolean>("anrEnabled") { options.isAnrEnabled = it }
       args.getIfNotNull<Boolean>("sendDefaultPii") { options.isSendDefaultPii = it }
       args.getIfNotNull<Boolean>("enableNdkScopeSync") { options.isEnableScopeSync = it }
+      args.getIfNotNull<Double>("tracesSampleRate") { options.tracesSampleRate = it }
```

Comment:
Is it needed for some reason? we should be able to track `AppStartState` even without that.

Reply:
Works without it with the current main branch. 👍